### DISCUSSION TITLE
Can I use 2 sub commands at once? I want to create a montage with all images increased in size

### DISCUSSION THREAD

**Discussion by ObiW4nKenobi** [ImageMagick/ImageMagick#7811] [Help]
### ImageMagick version

7.1.1-41

### Operating system, version and so on

Windows 10 23H2

### Description

I've successfully created a montage of jpeg images with this command:
`magick montage -adjoin *.jpg `in the PowerShell prompt.``
But now, I wanna be able to tell the program the final dimensions of my collages like 2560x1440 or 3840x2160. And how big each image in the collage should be. How could I make them smaller or larger for the collage?
So, I need to know how to scale each image individually and how to scale the final single file for the collage.
Thank You!

**Comment by fmw42** [comment]
The only thing that I know is to resize each image before hand and then resize the final result afterwards, at least the latter. You can pipe the result of magick montage to magick to do the final resize.

**Comment by ObiW4nKenobi** [comment]
Well, the thing is I obviously expect a minimum size for each thumbnail
image and then I want to put these into a row-column montage using Image
Magick. So, I think my best option is just to make a really big montage and
then I could maybe crop it at the end or just let the final size be and
scroll where I can't see. I guess experience is gonna be my best teacher
Here. I just installed this software yesterday and I sense it's power
already from the 2 montages I did create successfully. It's very easy to do
but maybe difficult to master. We shall see!

**Comment by snibgo** [comment]
See the montage documentation at https://imagemagick.org/script/montage.php

Doesn't "-geometry" do what you want, to set the size of each image?

**Reply by ObiW4nKenobi** [reply]
I have not yet tried that as I'm brand new to this application. I'll give it a shot. Thank You!
